File: scratch/extract_share.py
import re
import json

path = r"C:\Users\gurus\.gemini\antigravity-cli\brain\e41ed9a6-bc29-47cb-ac40-8d64de73995a\.system_generated\steps\3229\content.md"

with open(path, "r", encoding="utf-8") as f:
    html = f.read()

# Let's search for JSON data in script tags
# Usually, WIZ data or Bard initialization data is in JS variables.
# Let's search for strings matching common patterns or just search for anything containing "nav" or "mobile"

# Let's search for any strings that mention srikrishnarealestate or show nav styles
print("Searching for keywords...")
for m in re.finditer(r"sri|nav|mobile|icon|desktop", html, re.I):
    start = max(0, m.start() - 200)
    end = min(len(html), m.end() + 200)
    print(f"Match at {m.start()}: {html[start:end]}\n" + "-"*50)

File: scratch/extract_share.js
const fs = require('fs');

const path = "C:\\Users\\gurus\\.gemini\\antigravity-cli\\brain\\e41ed9a6-bc29-47cb-ac40-8d64de73995a\\.system_generated\\steps\\3229\\content.md";

if (!fs.existsSync(path)) {
  console.log("File does not exist: " + path);
  process.exit(1);
}

const html = fs.readFileSync(path, 'utf8');

console.log("Searching HTML content...");
// Let's search for "nav" or "mobile" or "icon" or "desktop"
const regex = /sri|nav|mobile|icon|desktop/gi;
let match;
const printed = new Set();

while ((match = regex.exec(html)) !== null) {
  const start = Math.max(0, match.index - 300);
  const end = Math.min(html.length, match.index + 300);
  const chunk = html.substring(start, end);
  
  // Clean up whitespace to make it readable
  const cleanChunk = chunk.replace(/\s+/g, ' ').substring(0, 400);
  
  if (!printed.has(cleanChunk)) {
    console.log(`Match at index ${match.index}:`);
    console.log(cleanChunk);
    console.log("-".repeat(50));
    printed.add(cleanChunk);
  }
  
  if (printed.size > 20) {
    console.log("Too many matches, stopping.");
    break;
  }
}
